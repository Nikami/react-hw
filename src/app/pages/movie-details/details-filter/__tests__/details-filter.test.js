import React from 'react';
import { shallow } from 'enzyme';
import DetailsFilter from '../index';

describe('DetailsFilter', () => {
  it('matches snapshot', () => {
    const defaultValue = 'test';
    const wrapper = shallow(<DetailsFilter genre={defaultValue} />);
    expect(wrapper).toMatchSnapshot();
  });
});
