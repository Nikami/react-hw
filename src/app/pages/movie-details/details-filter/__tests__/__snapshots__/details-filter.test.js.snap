// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailsFilter matches snapshot 1`] = `
<WithStyles(FilterContainer)>
  <div
    className="details-filter font-bold"
  >
    Films by 
    <span
      className="details-filter__genre"
    >
      test
    </span>
  </div>
</WithStyles(FilterContainer)>
`;
